schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -247,54 +258,72 @@
+                        "oneOf": [
+                            {
+                                "required": ["version", "status"],
+                                "maxProperties": 2
+                            },
+                            {
+                                "required": ["version", "status", "range", "limit"]
+                            }
+                        ],
                         "properties": {
-                            "versionGroup": {
-                                "type": "string",
-                                "description": "A string that represents a version branch, group, or a major version (e.g. 10.0, 3.1.*) where all version values are typically sequential or versionAffected comparisons are meaningful (optional).",
-                                "minLength": 1,
-                                "maxLength": 1024
+                            "version": {
+                                "description": "The single version being described, or the version at the start of the range.",
+                                "$ref": "#/definitions/version"
                             },
-                            "versionValue": {
+                            "status": {
+                                "description": "The vulnerability status for the version or range of versions. For a range, the status may be refined by the 'changes' list.",
+                                "$ref": "#/definitions/status"
+                            },
+                            "range": {
                                 "type": "string",
-                                "description": "The version name/value (e.g. 10.0.1, 3.1.2, \"IceHouse\")",
+                                "description": "The version numbering system used for specifying the range ('semver', 'git', and so on, as defined at TODO-URL). This defines the exact semantics of the comparison (less-than) operation on versions, which is required to understand the range itself.",
```

Comment:
Add an example?
examples: ["semver", "git", "patch", "rpm", "custom"]

I feel we should not make it an enum, to allow a CNA to use a constant value of their choice for their versioning scheme.

Reply:
Done, and agreed.